archai/nlp/nvidia_transformer_xl/onnx/onnx_utils/load.py

```diff
@@ -76,6 +76,11 @@ def load_from_pt(torch_model_path: str) -> Tuple[MemTransformerLM, dict]:
     checkpoint['model_config']['use_cache'] = True
 
     # Initializes the model
+
+    # Compatibility, TODO: Remove when not necessary
```

Comment:
Should this still be necessary since we are re-training every model from scratch? They should not have this key anymore, right?